samples/azure-quantum/chemistry/QuantumPhaseEstimation/Molecule.ipynb

```diff
@@ -0,0 +1,1775 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Simulate molecular ground state with quantum algorithms\n",
+    "\n",
+    "In this example, we'll simulate the molecular ground state of caffeine and hydrogen (H2) using Quantum Phase Estimation (QPE) and Variational Quantum Eigensolver (VQE). We'll show how many quantum resources (qubits, gates) the different approaches require and run the results on a quantum simulator.\n",
+    "\n",
+    "To read more about these two algorithms, take a look at our blog post on the Q# Dev blog: [Quantum developer tools for chemistry](https://devblogs.microsoft.com/qsharp/quantum-developer-tools-for-chemistry/)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "data": {
+      "application/json": "{\"LastUpdated\":\"2021-05-20T17:37:40.5328178-07:00\",\"IsCompleted\":false,\"Description\":\"Adding package microsoft.quantum.chemistry.jupyter\",\"Subtask\":null}",
+      "text/plain": [
+       "Adding package microsoft.quantum.chemistry.jupyter."
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/json": "{\"LastUpdated\":\"2021-05-20T17:37:42.1312255-07:00\",\"IsCompleted\":false,\"Description\":\"Reloading workspace\",\"Subtask\":null}",
+      "text/plain": [
+       "Reloading workspace."
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import qsharp\n",
+    "from qdk.chemistry import Molecule\n",
+    "from qdk.chemistry.broombridge import load_and_encode\n",
+    "qsharp.reload()\n",
```

Comment:
Similarly here, we should not need this line any longer to indicate dependencies from Q# workspaces on NuGet packages.

Reply:
I am not sure why but it doesn't work for me if I don't use `qsharp.reload()`.

Reply:
Let me take a look, then, as it sounds like there may be a bug, then. We shouldn't need that here if there's a csproj that includes a package reference and `IQSharpLoadAutomatically`, so if that's not working we may need to open an IQ# issue.

Reply:
I figured out why it didn't work, there was a missing semicolon somewhere in the Q# file so the compilation silently failed such that importing via Python did not work. Adding `qsharp.reload()` here is therefore is just to trigger compilation and expose any errors in the Q# program (ie. not silently fail).